web/public/templates/dojos/dojo-detail.dust

```diff
@@ -25,11 +25,13 @@
               </div>
             </div>
             <div cd-dojo-actions can-manage-dojo="isDojoAdmin" can-manage-events="isTicketingAdmin" dojo="dojo"></div>
-            <div cd-rounded-image src="https://s3-eu-west-1.amazonaws.com/zen-dojo-images/" src-id="{{ dojo.id }}" src-fallback="/img/avatars/dojo-default-logo.png" alt="{{ dojo.name }}" class="col-md-3 hidden-xs"></div>
-            <div class="col-md-6 cd-dojo-detail__dojo-name">
-              <h2 ng-bind="::dojo.name"></h2>
-              <h5 ng-if="dojo.website" class="cd-dojo-detail__dojo-website"><a ng-href="{{ dojo.website }}" title="{@i18n key="Website"/}"> <i class="fa fa-globe"></i> <span ng-bind="::dojo.website"></span></a></h5>
-            </div>
+              <div class="cd-profile__avatar">
+                <div cd-rounded-image src="{{ '/api/2.0/profiles/' + profile.id + '/avatar_img' }}" alt="{@i18n key="Avatar"/}" class="cd-profile__avatar"></div>
```

Comment:
To clarify, the change to the class is fine. Just need to use the original `src` and `alt`, and include the `src-id` and `src-fallback` from the original code too.